### DISCUSSION TITLE
A rev rework idea

### DISCUSSION THREAD

**Discussion by Contrabang** [ParadiseSS13/Paradise#19747] [Documents]
## Whats the problem with Revolutionaries?

Well they've been out of rotation for a while, but its generally that people go overboard and jump straight into mass murder. Which... is very hard to administrate. Its also the fact that revs can very silently snowball, and very quickly at that.

## My idea

Remove the flashes, headrevs can use pre-determined codewords to decrease the "morale" of the crew around them. This morale would be a variable determining how loyal the person is to Nanotrasen. At a certain point, say 0 morale, you become a revolutionary. Mindshielded personnel may also speak to crew around them, increasing their morale, and more if they're a high ranking role like Captain. Morale would also be modified by the circumstances around them, people's morale being more difficult to change if their cuffed. I know theres some Stockholm syndrome but it, but cmon, ya are being kidnapped by the opposing side. Things like megaphones or hell, standing on a table to get everyone's attention could also increase the percentage of morale changed. (I like to imagine the captain and headrev on opposite ends of a table arguing, trying to get the public on their side in the bar.)

This would make it very obvious if you're say trying to mass convert everyone in the bar by yelling out these codewords. But you could also be talking to someone alone in the library and more silently convert them. Mindshielding someone would give them an immediate morale boost, but also slowly and constantly increase their morale over time. So mindshielding people in processing would still work, but it wouldn't be instant.

Now something that I think is optional and would just be mostly interesting, is that being mindshielded would not stop the conversion process, but makes it much harder. So you could tie up a mindshielded person and convert them by yelling codewords at them, but because of their mindshield their morale would slowly improve and they would become de-rev'd. The only ones immune to actually being rev'd then would be the heads themselves. Anyone becoming a target of the revolution would be exempt from being a rev themselves.

I think this could make for some fun RP situations because they will actually HAVE to talk to people and deincentivize on the mass murder because the only ones that really have to die are the heads themselves (if the mindshield being optional change is included).

## Codewords
Now obviously this system wouldn't be perfect. Maybe there would be cooldowns on each word so you cant just copy and paste the same word over and over. Probably have decreasing returns also for how many codewords are in each message. It would require finetuning to get the exact numbers, but I feel like there is a good balance somewhere in there.

### Examples
Paycheck, Paycheque, Revolution, Viva, Credit, Cash, Dollar, Money, Union, Communism, Socialism, Capitalism, Dental, Healthcare, Wage, War-crime, Monopoly, Coup, Regime, Freedom, Proletariat, Bourgeoisie, King, Emperor, Queen, Overthrow, Theory, Reactionary, etc.
Random Year between 2100 and current year (so that people could guess that something bad nanotrasen did in that year)
Random command's names

## Wont be perfect

I still think this would be a heavy admin-required game mode, probably not something put into main rotation. But I think it would be a lot more fun than our current rev rounds of flash spamming and mass murder, if an admin does decide to do a rev round.

**Comment by mariadkgdmgdmi** [comment]
Actually sounds cool
